mne/viz/ica.py

```diff
@@ -400,7 +400,7 @@ def _plot_ica_sources_evoked(evoked, picks, exclude, title, show, labels=None):
     exclude_labels = list()
     for ii in picks:
         if ii in exclude:
-            line_label = 'ICA %03d' % (ii + 1)
+            line_label = 'IC #%03d' % (ii)
```

Comment:
thx, just fixed it in #3565 - can you reference that here, @jaeilepp? I'm not sure how or whether I can do that here.